tests/installation/user_settings.pm

```diff
@@ -40,7 +40,7 @@ sub run {
         my $max_tries = 4;
         my $retry     = 0;
         do {
-            $self->enter_userinfo(max_interval => utils::VERY_SLOW_TYPING_SPEED);
+            $self->enter_userinfo(max_interval => 1);
```

Comment:
I thought we didn't wanted to hide the bug.
A simpler name is already done as default.

This is for the test suite default, which has setting `ASSERT_BSC1122804` which uses the problematic full name so we are still able to reproduce the typing issue in this test suite.
(Similar to system_performance, where the failure was removed from all other tests to continue testing and still reproduce that bug in on scenario.

Reply:
I see. Well, as this is the code branch that should check for the bug and only apply a workaround *after* detecting the issue, then why even type slower than normal?

Reply:
I assume that even with max_interval=1, it will happen. This will help for:

1. Have a prove for the bug and the developer who would work on the bug what we have already tried.
2. Because once we have marked it as soft-failed, maybe after several slow retires we can continue the test.

Reply:
Hm, dunno . I leave it to you :)